app/geospatial_metadata.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+require 'active_record'
+require 'active_support/core_ext/hash'
+require 'yaml'
+require 'pry-byebug'
+require 'nokogiri'
+require 'rack'
+
+module GeospatialMetadata
```

Comment:
I'm still learning Rails tests, so maybe the pattern in this file is very common, but could we add a quick comment for this test? I gather that we're testing metadata here, but it's unclear to me what part of the process we're actually trying to evaluate.

Reply:
Are you referring to the test or the class? In general, the descriptions for the tests should be enumerated in the 'describe' and 'it' text for a test. You're right though that the class could maybe use a top line comment.